website/content/en/guides/level-up/log_namespace.md

```diff
@@ -0,0 +1,185 @@
+---
+title: The Log Namespace feature
+short: Log Namespace
+description: Learn how the log namespacing works.
+author_github: https://github.com/hoverbear
+domain: schemas
+weight: 2
+tags: ["log_namespace", "logs", "namespace", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand this feature and the global schema are mutually exclusive.
+* When log namespacing is enabled, the [global schema settings] are ignored.
+* This feature is still in `beta` so behavior might change.
+
+[global schema settings]: /docs/reference/configuration/global-options/#log_schema
+[docs.setup.quickstart]: /docs/setup/quickstart/
+{{< /requirement >}}
+
+## Default Behavior
+
+### Vector Config
+
+<details open>
+  <summary>Show/Hide</summary>
+
+```yaml
+schema:
+  log_namespace: true
+
+sources:
+  s0:
+    type: demo_logs
+    format: shuffle
+    lines:
+      - Hello World!
+    interval: 10
+
+transforms:
+  t0:
+    type: remap
+    inputs:
+      - s0
+    source: |
+      .message = .
+      .from = "t0"
+
+  t1:
+    type: remap
+    inputs:
+      - s0
+    source: |
+      .message = .
+      .from = "t1"
+
+sinks:
+  text_console:
+    type: console
+    inputs:
+    - t0
+    encoding:
+      codec: text
+
+  json_console:
+    type: console
+    inputs:
+    - t1
+    encoding:
+      codec: json
+      json:
+        pretty: true
+```
+
+</details>
+
+### stdout
+Sample output from `text_console`:
+
+```text
+{"from":"t0","message":"Hello World!"}
+```
+
+Sample output from `json_console`:
+
+```json
+{
+  "from": "t1",
+  "message": "Hello World!"
+}
+```
+
+## Difference with global schema
+
+If we switch this feature off:
+
+```yaml
+schema:
+  log_namespace: false
+```
+
+Then we will observe a big difference for these two encoders:
+- The `text` encoder only encodes the result of `log.get_message()`.
```

Comment:
I don't think Vector users will know what `log.get_message()` is or does 🤔  I think it'd be better to explain how the `text` encoder chooses the message from the user's point of view.